dessert/banana_muffins.md

```diff
@@ -0,0 +1,18 @@
+Ingredients:
+<li> 5 bananas
+<li> 1/2 cup apple sause
+<li> 3 tablespoons ground flaxseed
+<li> 1 teaspoon of vanilla extract
+<li> 1/2 teaspoon cinnamon
+<li> 3 cups milk
+<li> 2 cups oat flour
+<li> 2 cups fine cut oats
+<li> 1 cup thick cut oats
+<li> 1 teaspoon baking powder
+<li> 1 cup walnuts
+<li> 100gr chocolate, chopped <br>
+<br>
+1. Mash the bananas <br>
+2. Mix all ingredients <br>
```

Comment:
what is the order ??

Reply:
there is no order :) i would start with the bananas

Reply:
there is no order :) i would start with the bananas